js/desktopCapturer/getSources.js

```diff
@@ -67,7 +68,7 @@ function getSources(options, callback) {
     id = getNextId();
     ipcRenderer.send('ELECTRON_BROWSER_DESKTOP_CAPTURER_GET_SOURCES', captureWindow, captureScreen, updatedOptions.thumbnailSize, id);
 
-    return ipcRenderer.once('ELECTRON_RENDERER_DESKTOP_CAPTURER_RESULT_' + id, function(event, sources) {
+    ipcRenderer.once('ELECTRON_RENDERER_DESKTOP_CAPTURER_RESULT_' + id, function(event, sources) {
```

Comment:
do we even still need this code?  this code was copied from electron because there was no way to use via sandbox'ing - but maybe they addressed???

Reply:
@lneir Yes, it has been addressed https://github.com/electron/electron/commit/be0e3d9555984f23635498f0c8c204ae894db2d9 👍